docs/source/services/timelock_service/reverse-migration.rst

```diff
@@ -0,0 +1,140 @@
+.. _timelock-reverse-migration:
+
+Reverse Migration
+=================
+
+.. danger::
+
+   AtlasDB does not natively support reverse migrations. Improperly executing reverse migration from external timestamp
+   and lock services can result in **SEVERE DATA CORRUPTION**! Please contact the AtlasDB team before attempting a
+   reverse migration.
+
+If one wishes to downgrade from an external TimeLock Server to embedded timestamp and lock services, one can perform
+the inverse of the database migrations mentioned in :ref:`manual-timelock-migration`. It is also important to update the
+embedded timestamp bound to account for any timestamps issued since the original migration.
+
+The reverse migration process must be run offline (with no AtlasDB clients running) and consists of the following steps.
+Throughout this document, we assume the AtlasDB client you are trying to reverse-migrate is ``client`` .
+
+#. Shut down your AtlasDB clients.
```

Comment:
Should make this clearer that we are either talking about just the one client, or *all* clients (as per above comment)